migrationtools/databases.py

```diff
@@ -238,15 +253,67 @@ class SeedSBM(Database):
     def _setup_tables(self, meta):
         self.subscription = meta.tables['subscriptions_subscription']
         self.messageset = meta.tables['contentstore_messageset']
+        self.schedule = meta.tables['contentstore_schedule']
         self.user = meta.tables['auth_user']
 
+    def lookup_messageset_with_name(self, name):
+        statement = select([self.messageset])\
+            .where(self.messageset.c.short_name == name)
+        return self.execute(statement).fetchone()
```

Comment:
messageset.name has a unique constraint on the column, so it will always be one or none, enforced by the DB.